vulnerability/scanner.go

```diff
@@ -126,14 +126,21 @@ func (f VulnerabilityScanner) scan(ctx context.Context, snap ec2.EBSSnapshot) {
 			Severities: []db_types.Severity{0, 1, 2, 3, 4},
 		},
 	}
-
+	now := time.Now()
 	report, err := f.runner.ScanVM(ctx, opts)
+	f.log.Debugf(
```

Comment:
should we change it to debug? will we have any indication that snapshots were scanned?

Reply:
Didn't want to spam the logs with information that will be relevant only when we want to debug.
If you think it's critical to the user I'll change to info.

Reply:
@jeniawhite Since you initially set it to `info`, what are your thought about this, will it spam the logs? how frequently will we see this log? Is it necessary for the first step of debugging (before asking users to switch log levels)

Reply:
The start scanning log is Info level (in the beginning of the func), so either both will be info or debug.

Reply:
Changed to info.
I changed my opinion it will be helpful to the user.